config/src/comments.rs

```diff
@@ -210,11 +210,40 @@ fn comments() -> HashMap<String, String> {
 		.to_string(),
 	);
 
+	retval.insert(
+		"i2p_mode".to_string(),
+		"
+#Configuration for I2P support (\"Disabled\" by default)
+#I2P can be enabled by the following:
+#[server.p2p_config.i2p_mode]
+#mode = \"Enabled\"
+
+#[server.p2p_config.i2p_mode.i2p_config]
+#autostart = true/false  - start I2P automatically
```

Comment:
all configuration options here will not be valid if uncommented, maybe try to make it as simple to get started when uncommenting them so:
```toml
# start I2P automatically
# autostart = true
# exclusively connect through I2P, or also use TCP/IP
# exclusive = false
```